linera-base/src/crypto/secp256k1.rs

```diff
@@ -187,6 +192,120 @@ impl fmt::Debug for Secp256k1PublicKey {
     }
 }
 
+impl From<Secp256k1PublicKey> for Owner {
+    fn from(value: Secp256k1PublicKey) -> Self {
+        Self(CryptoHash::new(&value))
+    }
+}
+
+impl From<&Secp256k1PublicKey> for Owner {
+    fn from(value: &Secp256k1PublicKey) -> Self {
+        Self(CryptoHash::new(value))
+    }
+}
+
+impl<'de> BcsHashable<'de> for Secp256k1PublicKey {}
+
+impl WitType for Secp256k1PublicKey {
```

Comment:
Can't the Witty types be derived for the two key types?

Reply:
TBH I just did the same thing that was already done for Ed25519.

Reply:
We should find out why it was done explicitly there. Or was that even before the `derive` macros for the Witty types were implemented?